dev-tools/packaging/templates/docker/Dockerfile.tmpl

```diff
@@ -30,11 +30,24 @@ RUN microdnf -y --setopt=tsflags=nodocs update && \
 {{- else }}
 RUN yum -y --setopt=tsflags=nodocs update \
   {{- if (eq .BeatName "heartbeat") }}
+    && PACKAGES_TO_INSTALL="atk cups gtk gdk xrandr ipa-gothic-fonts xorg-x11-fonts-100dpi xorg-x11-fonts-75dpi xorg-x11-utils \
+xorg-x11-fonts-cyrillic xorg-x11-fonts-Type1 xorg-x11-fonts-misc" \
+    && case "$(arch)" in \
+         x86_64) \
+            PACKAGES_TO_INSTALL="$PACKAGES_TO_INSTALL pango.x86_64 libXcomposite.x86_64 libXcursor.x86_64 libXdamage.x86_64 \
+libXext.x86_64 libXi.x86_64 libXtst.x86_64 cups-libs.x86_64 libXScrnSaver.x86_64 libXrandr.x86_64 GConf2.x86_64 \
+alsa-lib.x86_64 atk.x86_64 gtk3.x86_64" \
```

Comment:
I wonder why we need to specify the architecture in these packages for amd64 but not for arm64. Do we actually need this? If not this would be simplified.

Reply:
I think we can  remove this. These are all dependencies for  chromium, which is used for synthetics. While synthetics probably won't work out of the box on arm till figure out how to get arm chromium builds there's no reason not to remove the arch from these packages.

Reply:
By remove, btw, I don't mean the packages, but the arch suffix